app/src/main/java/com/jetbrains/handson/mpp/mobile/MainActivity.kt

```diff
@@ -47,7 +51,8 @@ class MainActivity : AppCompatActivity(), ApplicationContract.View,
         button.setOnClickListener {
             val origin = outboundSpinner.selectedItem.toString()
             val destination = inboundSpinner.selectedItem.toString()
-            presenter.onButtonPressed(origin, destination)
+            val time = LocalDateTime.now().plusMinutes(5).withNano(0).toString()
```

Comment:
I imagine 'toString' can be called inside onButtonPressed. In fact, this whole 'LocalDateTime' part can be called there, since I think it won't be android-specific :)